app/src/main/java/com/example/travenor/screen/onboarding/UserInterestSurveyPresenter.kt

```diff
@@ -0,0 +1,34 @@
+package com.example.travenor.screen.onboarding
+
+import com.example.travenor.constant.Food
+import com.example.travenor.constant.Place
+import com.example.travenor.data.sharedpreference.SharedPreferencesManager
+import java.io.IOException
+
+class UserInterestSurveyPresenter internal constructor(private val sharedPrefsManager: SharedPreferencesManager) :
```

Comment:
```suggestion
class UserInterestSurveyPresenter internal constructor(private val sharedPrefsManager: SharedPreferencesManager) :
```
Không pass trực tiếp ``SharedPreferencesManager`` vào Presenter mà thông qua ``Repository``
Trong trường hợp này là ``UserRepository``